lib/interfaces.ts

```diff
@@ -11,6 +11,7 @@ export interface IMapFactory {
   (stringOrArray: string | string[]): IMapping;
   map(stringOrArray: string | string[]): IMapping;
   execute(source?, destination?);
```

Comment:
Nope. That was there for compatibility before we made the breaking change. I'll fix.